### PR TITLE
feat(examples): initial rsc example

### PR DESCRIPTION
**Pull request by notrab**
# Description

Please include a summary of the change and which issue is fixed.
Please also include relevant motivation and context.

# Type of change

- [ ] 💔 Breaking
- [ ] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [ ] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

File: examples/nextjs-rsc/app/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+const RootLayout = async ({ children }: { children: React.ReactNode }) => {
+  const { postCollection } = await grafbase.request(GetAllPostsDocument, {
+    first: 50
+  })
+
+  return (
+    <html lang="en">
+      <head>
+        <title>Grafbase + Next 13</title>
```
Comment (fbjork) [inline examples/nextjs-rsc/app/layout.tsx:30]:
```suggestion
        <title>Grafbase + Next.js 13</title>
```
```suggestion
        <title>Grafbase + Next 13</title>
```

File: examples/nextjs-rsc/app/page.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <>
+      <h1>Next.js 13 + Grafbase</h1>
+      <p>
+        Here you can select a post from the menu on the left and open it inside
+        of a nested route.
+      </p>
+      <h3>You will need to</h3>
+      <ol>
+        <li>
+          Go to <a href="http://localhost:4000">http://localhost:4000</a>
```
Comment (fbjork) [inline examples/nextjs-rsc/app/page.tsx:12]: Use relative path here so it works in prod?

Comment (notrab) [reply examples/nextjs-rsc/app/page.tsx:12]: We’d need to switch to use a NEXT_PUBLIC environment variable. I can do this.

Comment (notrab) [reply examples/nextjs-rsc/app/page.tsx]: I've updated the text here to be more generic @fbjork

**Review by jvidalv — COMMENTED** [review]
Looks good! Maybe we could add pagination/load more for v2? Would be an interesting issue to tackle on this new paradigm!

**Pull request merged**